### ISSUE TITLE
Sample JSON ingest page incorrect.

### ISSUE THREAD

**Issue by pflickin** [MicrosoftDocs/dataexplorer-docs#1122]
The sample JSON ingest page for Python appears to have several inconsistencies with the Python ingest page (https://docs.microsoft.com/en-us/azure/data-explorer/python-ingest-data). For instance, there appears not to be a KustoIngestClient, so the examples do not work. Additionally, the examples are very hard to follow given that sample data is only provided for the first and second examples with the user left guessing on how to format the multijson example.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 68d5f8ab-85c7-19bf-d91a-e76e81c76d7b
* Version Independent ID: 161a10d8-fc7f-97e8-b13e-0ddc7dd4f3c6
* Content: [Ingest JSON formatted data into Azure Data Explorer](https://docs.microsoft.com/en-us/azure/data-explorer/ingest-json-formats?tabs=python)
* Content Source: [data-explorer/ingest-json-formats.md](https://github.com/MicrosoftDocs/dataexplorer-docs/blob/main/data-explorer/ingest-json-formats.md)
* Service: **data-explorer**
* GitHub Login: @shsagir
* Microsoft Alias: **shsagir**

**Comment by CHEEKATLAPRADEEP-MSFT** [comment]
@pflickin  Thanks for the feedback! I have assigned the issue to the content author to evaluate and update as appropriate.

**Comment by shsagir** [comment]
@pflickin Thanks so much for taking the time to leave us your comment on our documentation. We have a [backlog work item](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1910703) to update the topic and have added your feedback to it.

We're closing this as an open issue for now, but feel free to continue to comment here as necessary. We appreciate your contribution to our community.

#please-close
